infoflow/entropy_kde.py

```diff
@@ -0,0 +1,86 @@
+import numpy as np
+from infoflow import multidimkde as mdkde
+from math import log
+
+
+def kerneldensityestimation(source, target, timelagx, timelagy, n, bw_coeff):
```

Comment:
According to PEP8, function names should be in the form ``kernel_density_estimation``.  That also goes for ``timelag_x`` and ``timelag_y``.